docs/about/history.md

```diff
@@ -0,0 +1,190 @@
+> have friendly conversations as we are both part of the "Redis ecosystem", but
+> I did not received any prior question about this issue.
+
+The following year, 2014, Garantia Data [changed its name to Redis Labs][redislabs].
+
+[redislabs]: https://techcrunch.com/2014/01/29/database-provider-garantia-data-makes-another-name-change-this-time-to-redis-labs/)
+
+In 2015, Salvatore left Pivotal for Redis Labs. He writes in his blog post
+[Thanks Pivotal, Hello Redis Labs](http://antirez.com/news/91):
+
+> Redis Labs was willing to continue what VMware and Pivotal started. I'll be
+> able to work as I do currently, spending all my time in the open source side
+> of the project, while Redis Labs continues to provide Redis users with an
+> hassles-free Redis experience of managed instances and products.
+
+In 2018, Redis Labs changed the license of some of its modules from AGPL to a
+source-available license. The license prevents competing cloud providers from
+offering these modules to customers and does therefore not fulfil the creteria
+for an open source license.
+
+This was interpreted by some as if Redis is no longer open source. Sanfilippo
+clarified on his blog [Redis will remain BSD licensed][antirez-remain-bsd].
+
+Yiftach Shoolman, CTO and co-founder of Redis Labs, also clarified in in the
+company's blog that [Redis’ License is BSD and will remain
+BSD][will-remain-bsd]. He repeated this promise in [a comment on Hacker
+News][hn-always-bsd], writing “let me assure you that Redis remains and always
+will remain, open source, BSD license”.
+
+[antirez-remain-bsd]: http://antirez.com/news/120
+[will-remain-bsd]: https://redis.io/blog/redis-license-bsd-will-remain-bsd/
+[hn-always-bsd]: https://news.ycombinator.com/item?id=17819392
+
+Also in 2018, Redis Labs aquired the trademark rights to the name Redis and the
+logo from Sanfilippo.
+
+In 2020, Salvatore Sanfilippo announced in his blog post [The end of the Redis
+adventure](http://antirez.com/news/133) that he is stepped back as the Redis
+maintainer, handing over the maintenance to Yossi Gottlieb and Oran Agra at
+Redis Labs. The two created a “core team” to maintain the project and invited
+Itamar Haber from Redis Labs, Zhao Zhao from Alibaba and Madelyn Olson from
+Amazon. The members were selected “based on demonstrated, long-term personal
+involvement and contributions”. This was described in the projects
+[Governance][governance-2020] page which was the inspiration for the current
+[Valkey governance](valkey-governance).
+
+[governance-2020]: https://web.archive.org/web/20200709170526/https://redis.io/topics/governance
+[valkey-governance]: https://github.com/valkey-io/valkey/blob/unstable/GOVERNANCE.md
+
+In 2021, the Redis Labs changed its name to Redis.
+
+The end of open source Redis
+----------------------------
+
+Despite the seemingly community-based core team, Redis (the company) maintained
+a majority in the core team as well as ownership of the Redis trademark. In
```

Comment:
```suggestion
Despite the seemingly community-based core team, Redis ltd. maintained ownership of the Redis trademark and control of the repositories. In
```

The fact they had a majority in the core team is, ultimately, irrelevant. They always owned the trademark and repositories and could simply change the license. They did meddle a lot, they delayed 7.2 for months, but not sure that is worth calling out here.

Reply:
Right, it's irrelevant. I'll just remove this part.